camera_to_hls/README.md

```diff
@@ -0,0 +1,52 @@
+# Membrane Demo - Camera Video to HLS
+
+This project demonstrates capturing camera video and converting it to HLS stream, which is then can be used wherever you want :)
+
+## Prerequisites
+
+You must have following packages installed on your system:
+
+- ffmpeg 4.\*
+
+### If using MACOS and `homebrew`:
+
+```shell
+brew install ffmpeg
+```
+
+## Run the demo
+
+Run the project with:
+
+```bash
+mix run --no-halt
+```
+
+HLS header and segment files will be created in `output` directory along with playlist files.
```

Comment:
That's incorrect. HLS only defines a playlist. Headers and segments are CMAF. If you want to know the boundary of HLS and what is something else, checkout [RFC 8216](https://datatracker.ietf.org/doc/html/rfc8216)